frontend/src/pages/main/home/client/components/graph/Graph.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {useEffect, useState} from 'react';
+import {Card, CardContent, Grid, Typography} from "@mui/material";
+import "../../../../../../common/styles/container.css";
+import "./styles.css";
+import "../balance/styles.css";
+import {Pie} from "react-chartjs-2";
+import {ArcElement, Chart, Legend, Tooltip} from "chart.js";
+import {Content} from "../../../../../../common/constansts/content";
+import {ITransaction} from "../../../../../../types/transactions";
+import {getTransactionsInRange} from "../../../../../../services/transaction";
+import {useAppSelector} from "../../../../../../stores/redux/hooks";
+import {selectLoggedUser} from "../../../../../../stores/redux/loggedUser/loggedUserSlice";
+import FinancialSummary from "../financialSummary/FinancialSummary";
+import {getGraphData} from "./utils";
+
+Chart.register(Tooltip, Legend, ArcElement);
+
+const Graph: React.FC = () => {
+
+    const loggedUser = useAppSelector(selectLoggedUser);
+    const [transactions, setTransactions] = useState<ITransaction[]>([]);
+    const [income, setIncome] = useState<number>(0);
+    const [expenses, setExpenses] = useState<number>(0);
+
+    const getTransactionData = async () => {
+        const transactionsData = await getTransactionsInRange(loggedUser.id, loggedUser.userType, new Date(), new Date());
```

Comment:
pass correct dates

Reply:
if you mean format like "dd.mm.yyyy" it's string, but i need to provide Date, so idk what should i do.

Reply:
Your start date must be 1 at January